lib/pages/views/subscriptionsList.dart

```diff
@@ -0,0 +1,194 @@
+import 'package:flutter/material.dart';
+import 'package:gradient_widgets/gradient_widgets.dart';
+import './common/CustomShapeClipper.dart';
+import './activityDetail.dart';
+
+class SubscriptionsList extends StatelessWidget {
+  const SubscriptionsList({Key key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return new Scaffold(
+      appBar: new AppBar(
+        elevation: 0,
+        // title: new Text('Palestras e Atividades'),
+        backgroundColor: Colors.red,
+      ),
+      body: Column(
+        children: <Widget>[
+          SubscriptionsListTopPart(),
+          ActivityFilter(),
+          SubscriptionsListContent(),
+        ],
+      ),
+    );
+  }
+}
+
+class SubscriptionsListTopPart extends StatefulWidget {
+  @override
+  _SubscriptionsListTopPartState createState() => _SubscriptionsListTopPartState();
+}
+
+class _SubscriptionsListTopPartState extends State<SubscriptionsListTopPart> {
+  String title = "Inscrições";
+
+  @override
+  Widget build(BuildContext context) {
+    return Stack(
+      children: <Widget>[
+        ClipPath(
+          clipper: CustomShapeClipper(),
+          child: Container(
+            height: 150.0,
+            color: Colors.red,
+            child: Column(
+              children: <Widget>[
+                SizedBox(
+                  height: 5.0,
+                ),
+                Padding(
+                  padding: const EdgeInsets.fromLTRB(50.0, 10.0, 0.0, 10.0),
+                  child: Row(
+                    children: <Widget>[
+                      Text(
+                        title,
+                        style: TextStyle(
+                          fontWeight: FontWeight.w800,
+                          fontSize: 24.0,
+                          color: Colors.white,
+                          fontFamily: 'Raleway',
+                        ),
+                      ),
+                      Align(
+                        alignment: Alignment.topRight,
+                        child: Hero(
+                          tag: "hero",
+                          child: Container(
+                            padding: EdgeInsets.only(top: 50.0),
+                            height: 80.0,
+                            width: 80.0,
+                            // child: logo,
+                          ),
+                        ),
+                      ),
+                    ],
+                  ),
+                )
+              ],
+            ),
+          ),
+        )
+      ],
+    );
+  }
+}
+
+class SubscriptionsListContent extends StatelessWidget {
```

Comment:
Eu fiz uma tela de inscrições que já está funcionando com o backend. Está na branch master... Eu fiz uma lista simples com Cards, depois podemos resolver os conflitos juntos e ver caso voce tenha feito alguma alteração visual para integrar com a parte de backend que fiz.

Reply:
Na branch master a tela que eu fiz com o backend funcionando é a view com o nome `subscriptionPage`... Ai a gente pode comparar as mudanças depois

Reply:
Ok, em casa eu vou dar uma olhada, meu plano era deixar igual a de listagem de atividades, mas teria um botão de cancelar inscrição no mesmo card, pra ficar mais prático, porém to com um problema de renderização nessa tela. Se der muito problema acredito que seja melhor só deixar o botão de se desinscrever dentro do detalhe da atividade mesmo

Reply:
Eu fiz mesmo parecido com a listagem de atividades, só não tem esse botão de cancelar no mesmo card.. Mas da para incluir... Não é difícil se tratanto de backend. Podemos tentar ver e fazer juntos depois... Dentro do detalhe da atividade já tem.

Reply:
Sim, o problema foi só que quando eu deixei o ListTile e o Button na mesma Row, sumiu tudo de dentro do card...